pyServer/manifests/linux/linux-repoconfig

```diff
@@ -0,0 +1,15 @@
+echo,### Probing Directories ###
+ll,/etc/products.d/
+ll,/etc/zypp/repos.d/
+ll,/etc/yum.repos.d/
+ll,/etc/apt
+
+echo,### Gathering Repository Configuration Files ###
+copy,/etc/products.d/*.prod,noscan
+copy,/etc/zypp/repos.d/*.repo,noscan
+copy,/etc/yum.repos.d/*.repo,noscan
+copy,/etc/apt/sources.list,noscan
+copy,/etc/apt/sources.list.d/*.list,noscan
+echo,
```

Comment:
Why do we need this "echo"?

Reply:
I'm not sure, but it's on every other configuration block so I duplicated it here - in fact, I missed it in the bootconfig.  I'll fix that.